staging/src/k8s.io/kube-aggregator/pkg/controllers/autoregister/autoregister_controller.go

```diff
@@ -169,29 +197,61 @@ func (c *autoRegisterController) processNextWorkItem() bool {
 	return true
 }
 
-func (c *autoRegisterController) checkAPIService(name string) error {
+// checkAPIService syncs the current APIService against a list of desired APIService objects
+//
+// Current:           \          Desired: | A. not found | B. sync on start | C. sync always
```

Comment:
I must be misunderstanding this table.  I expected to see `current \ desired` in the first column, but nothing has a backslash and most don't have two conditions.  Is this column just "observed state"?